### PR TITLE
Introduce PAUSER privilege levels

### PR DESCRIPTION
**Pull request by jhand2**
Differentiate PAUSERs by privilege levels. This specification calls out that there can only be one High privilege PAUSER. The integration spec should go into more detail about this.

In particular, this change restricts PAUSERs at PL1 from calling CertifyKey.

### CODE REVIEW

File: runtime/README.md [hunk 1]
```diff
@@ -270,6 +270,17 @@ this case, the new Runtime Firmware must:
 Caliptra Runtime Firmware SHALL implement a profile of the DICE Protection
 Environment (DPE) API.
 
+### PAUSER Privilege Levels
+
+Caliptra models PAUSER callers to its mailbox as having 1 of 2 privilege levels:
+
+* PL0 - High Privilege. Only 1 PAUSER in the SoC may be at PL0. The PL0 PAUSER
+  is denoted in the signed Caliptra firmware image. The PL0 PAUSER may call any
+  supported DPE commands.
```
Comment (andreslagarcavilla) [inline runtime/README.md:279]: Only PL0 can use the CertifyKey command. Availability of CertifyKey output evidence signifies the PL0 status of the consumer

(or something like that)

**Review by andreslagarcavilla — DISMISSED** [review]

File: runtime/README.md (hunk 1 above)
Comment (jhand2) [reply runtime/README.md]: Done

**Review by varuns-nvidia — DISMISSED** [review]

**Review by FerralCoder — APPROVED** [review]

**Comment by jhand2** [comment]
Sorry @varuns-nvidia @FerralCoder, I accidentally dismissed reviews with a rebase.

**Review by bluegate010 — APPROVED** [review]

**Review by andreslagarcavilla — APPROVED** [review]

**Comment by varuns-nvidia** [comment]
I'll approve again, but noting that we need to track the corresponding manifest changes. I don't see a field in https://github.com/chipsalliance/caliptra-sw/tree/main/rom/dev#81-firmware-manifest for marking the PL0 PAUSER.

**Review by varuns-nvidia — APPROVED** [review]

**Comment by jhand2** [comment]
> I'll approve again, but noting that we need to track the corresponding manifest changes. I don't see a field in https://github.com/chipsalliance/caliptra-sw/tree/main/rom/dev#81-firmware-manifest for marking the PL0 PAUSER.

Thanks. I've opened https://github.com/chipsalliance/caliptra-sw/issues/182 to track

**Pull request merged**